### PR TITLE
completed variable add component and variable show component

### PR DESCRIPTION
**Pull request by upexcel**

### CODE REVIEW

File: src/app/modules/add-var/add-var.component.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+})
+export class AddVarComponent implements OnInit {
+    ckeditorContent: any;
+    types: number;
+    type: any;
+    constructor(public dialogRef: MdDialogRef<any>, private getVariable: ImapMailsService) {
+    }
+
+    ngOnInit() {
+        this.types = 1;
```
Comment (reactexcel) [inline src/app/modules/add-var/add-var.component.ts]: use constant

File: src/app/modules/email-variables/email-variables.component.html [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </md-card-content>
+	</md-card>
+</div>
+<div class="header">
+	<md-card>
+	<h3>System Variable(s)</h3>
```
Comment (reactexcel) [inline src/app/modules/email-variables/email-variables.component.html]: make this a different componet

File: src/app/modules/add-var/add-var.component.ts (hunk 1 above)
Comment (upexcel) [inline src/app/modules/add-var/add-var.component.ts]: tried constant in place of types number getting error "A class member cannot have the 'const' keyword"

File: src/app/modules/email-variables/email-variables.component.html (hunk 2 above)
Comment (upexcel) [inline src/app/modules/email-variables/email-variables.component.html]: right now System variable is just a simple list. there is no functionality for system variable in HR system.

**Pull request closed without merging**